other.yml

```diff
@@ -52,3 +54,13 @@ events:
         - 2023-01-02 15:05:00
         - 2023-02-20 15:05:00
         - 2023-03-20 15:05:00
+  - summary: "Lean SCS Operator Coffee"
```

Comment:
If you start a new series on 2023-04-24, shouldn't the end date of the old series be adjusted to not overlap?